### ISSUE TITLE
New links not working

### ISSUE THREAD

**Issue by kjettern** [meganz/MEGAcmd#343]
The new link structure in mega does not work and gives an error. 
Hopefully someone have a quick fix, also for my Synology NAS.

The old one was http://mega.nz/#F!XXXXXXXX
But the new one that's not working is: https://mega.nz/folder/XXXXXXXX

**Comment by polmr** [comment]
Windows & MacOS last version from https://mega.nz/cmd already support those links. For linux, updating `megacmd` should also work too. We are currently working on a new version for Synology NAS. In the meantime, you might want to translate the links with:
```echo place_your_link_here | sed "s/#/\!/g" |  sed "s#folder/#\#F\!#g" | sed "s#file/#\#\!#g"```

**Comment by bloodmaan** [comment]
Why you lie ?????
No update for arm Linux

**Comment by polmr** [comment]
You are correct @bloodmaan . Raspbian version is dragging. We will provide updates. Thanks for mentioning and apologies for the misguiding assertion
